### PR TITLE
modernize the readme a bit (areas: IDEs, bootstrapping)

### PR DESCRIPTION
**Pull request by SethTisue**
context: at https://contributors.scala-lang.org/t/scala-build-process/3624/4 , @tfinneid aspires to add Scala to debian

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,24 +159,15 @@ Assume the current `starr` version is `2.12.0` (defined in
...
 ### IDE setup
 
-You may use IntelliJ IDEA (see [src/intellij/README.md](src/intellij/README.md)),
-or ENSIME (see [this page on the ENSIME site](https://ensime.github.io/editors/)).
-(Support for Eclipse has decayed and been removed, but could be resurrected by
-a volunteer.)
+We suggest using IntelliJ IDEA (see
+[src/intellij/README.md](src/intellij/README.md).
+
+(Metals should also work, but we don't yet have instructions or sample
```
Comment (hrhino) [inline README.md]: I'm sure most people who get this far have heard of it, but a metals link wouldn't hurt (and ENSIME got one!)

**Review by hrhino — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -127,7 +127,7 @@ Once you've started an `sbt` session you can run one of the core commands:
       to skip generating / publishing API docs (speeds up the process).
 
 If a command results in an error message like `a module is not authorized to depend on
-itself`, it may be that a global SBT plugin (such as [ENSIME](https://ensime.github.io/)) is
+itself`, it may be that a global SBT plugin is
```
Comment (Philippus) [inline README.md]:
```suggestion
itself`, it may be that a global sbt plugin is
```

File: README.md [hunk 3]
```diff
@@ -127,7 +127,7 @@ Once you've started an `sbt` session you can run one of the core commands:
...
 resulting in a cyclical dependency. Try disabling global SBT plugins (perhaps by
```
Comment (Philippus) [inline README.md]:
```suggestion
resulting in a cyclical dependency. Try disabling global sbt plugins (perhaps by
```

Comment (hrhino) [reply README.md]: They should have kept it as an acronym for "SBT Build Tool".

**Review by diesalbla — APPROVED** [review]
Goodbye Eclipse and Ensime. 😐

**Pull request merged**